private/webpack.config.js

```diff
@@ -1,200 +1,74 @@
-  const staticPlugins = [
-    new CopyWebpackPlugin({
-      patterns: [
-        {
-          context: SRC_PATH,
-          from: 'static/**/*',
-          to({ absoluteFilename }) {
-            return absoluteFilename.replace(`${SRC_PATH}/static`, OUT_PATH);
-          },
-        },
-      ],
-    }),
-  ];
-
-  // if we're only building static assets, skip CSS extraction
-  if (env?.entry === 'static') {
-    return [
-      new ESLintPlugin({ extensions: ['js', 'jsx', 'ts', 'tsx'] }),
-      // Sets mode so we can access it in `postcss.config.js`.
-      new webpack.LoaderOptionsPlugin({ options: { mode: argv.mode } }),
-      new StyleLintPlugin({ threads: true }),
-      ...staticPlugins,
-    ];
-  }
-
-  const plugins = [
-    new ESLintPlugin({ extensions: ['js', 'jsx', 'ts', 'tsx'] }),
-    // Sets mode so we can access it in `postcss.config.js`.
-    new webpack.LoaderOptionsPlugin({ options: { mode: argv.mode } }),
-    // Extract CSS to own bundle, filename relative to output.path.
-    new MiniCssExtractPlugin({ filename: '../styles/[name].css', chunkFilename: '[name].css' }),
-    new StyleLintPlugin({ threads: true }),
-  ];
-
-  // we're specifying an entry point that isn't static, skip static processing
-  if (env?.entry) {
-    return plugins;
-  }
-
-  // do everything
-  return [...plugins, ...staticPlugins];
-};
-
-/**
- * Get the cache configuration for the build mode
- * @param {String} mode the build mode
- * @returns {Object|False}
- */
-const getCacheConf = (mode) => {
-  if (mode === 'production') {
-    return false;
-  }
-
-  return {
-    type: 'filesystem',
-    profile: false,
-    buildDependencies: {
-      config: [__filename],
-    },
-  };
-};
-
-const config = (env, argv) => ({
-  bail: argv.mode === 'production',
-  cache: getCacheConf(argv.mode),
-  target: 'web',
-  profile: false,
-  devtool: 'source-map',
-  entry: getEntries(env),
+  },
   output: {
     filename: '[name].js',
     path: path.resolve(__dirname, `${OUT_PATH}/scripts`),
-    pathinfo: false,
   },
-  externals: {
-    lodash: 'lodash',
-    react: 'React',
-    'react-dom': 'ReactDOM',
-    '@wordpress/i18n': 'wp.i18n',
-  },
-  watchOptions: {
-    ignored: /node_modules/,
-  },
-  performance: {
-    hints: false,
-  },
-  optimization: {
-    minimize: argv.mode === 'production',
-    minimizer: [
-      new EsbuildPlugin({
-        target: 'es2015',
-        css: true,
-      }),
-    ],
-  },
-  resolve: {
-    symlinks: false,
-  },
-  stats: {
-    all: false,
-    assets: true,
-    errors: true,
-    errorDetails: true,
-    excludeAssets: [/\.(eot|ttf|woff2?|jpg|png|svg)$/],
-  },
-  plugins: getPlugins(argv, env),
+  plugins: [
+    ...defaultConfig.plugins
+      .map((plugin) => {
+        if (plugin instanceof CopyWebpackPlugin) {
+          return new CopyWebpackPlugin({
+            patterns: [
+              ...plugin.options.patterns,
+              {
+                from: 'static/**/*',
```

Comment:
This is not copying the static files as expected, I haven't been able to debug why.